week2/Task9/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿// 9. Ganymede calls
+var time = new JupiterTime  (2,70);
+var time1 = new TitanTime (9,50);
+var time2 = new GanymedeTime (8,80);
+Console.WriteLine(time);
+Console.WriteLine(time1);
+Console.WriteLine(time2);
+
+public abstract class AlienTime
+{
+    public int Hours { get; set; }
+    public int Minutes { get; set; }
+    private int HoursInDay { get; set; }
+
+    public AlienTime(int hours, int minutes, int hoursInDay)
+    {
+        Hours=hours;
+        Minutes=minutes;
+        HoursInDay=hoursInDay;
+    }
+
+    public override string ToString()
+    {
+        var NumberOfDays=HoursInDay/171;
```

Comment:
variable names should start with lowercase, `var numberOfDays = `

Reply:
Done